fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/impl/networking/server/ServerPlayNetworkAddon.java

```diff
@@ -76,11 +76,6 @@ public void onClientReady() {
 	 * @return true if the packet has been handled
 	 */
 	public boolean handle(CustomPayloadC2SPacket packet) {
-		// Do not handle the packet on game thread
-		if (this.server.isOnThread()) {
-			return false;
-		}
-
```

Comment:
This is not needed to fix the issue at hand, but I thought I might as well do it for consistency and in case bundled C2S packets are ever introduced. Can be reverted if desired.